__docs__/wonder-blocks-switch/switch.stories.tsx

```diff
@@ -82,46 +82,50 @@ export const Default: StoryComponentType = {
  * The `onChange` prop is optional in case the toggle will
  * be wrapped in a larger clickable component.
  */
-export const Controlled: StoryComponentType = () => {
-    const [checkedOne, setCheckedOne] = React.useState(false);
-    const [checkedTwo, setCheckedTwo] = React.useState(false);
+export const Controlled: StoryComponentType = {
```

Comment:
**note:** This component doesn't have any big changes, just moved the code around to support CSF3.